lib/wca_exceptions.rb

```diff
@@ -32,4 +32,23 @@ def initialize(error_str)
       super(:forbidden, error_str)
     end
   end
+
+  class RegistrationError < ApiException
+    attr_reader :error_code, :error, :data
+
+    def initialize(status, error, data = nil)
+      super(status, I18n.t("competitions.registration_v2.errors.#{error}"))
+      @error = error
+      @data = data
+    end
+  end
+
+  class BulkUpdateError < StandardError
+    attr_reader :status, :errors
+
+    def initialize(status, errors)
+      @status = status
```

Comment:
why did you not super here?

Reply:
I can't remember the exact chain of reasoning, but because `errors` is an array not a string it produces an error if I call super (as you can't call super without arguments). This is the same workaround I used in the microservice, but we can tweak the design a bit more if needed